front/app/components/FormBuilder/components/FormBuilderSettings/utils.tsx

```diff
@@ -35,6 +36,21 @@ export function getAdditionalSettings(
 
   switch (field.input_type) {
     case 'multiselect':
+      return (
```

Comment:
Perhaps out of scope for this PR, but maybe we should discuss this this pattern of defining functions that return JSX without being components in the next CDM. In theory a functional component is also a function that returns JSX, so they look very similar, and React tricks you into thinking that you can treat it (React) as just JavaScript when you can't. But there are actually some consequences to calling a function that returns JSX vs using a component- see [here](https://stackoverflow.com/a/71660376/7237112). The most important one is that for components, the virtual dom can do optimizations which means that it can avoid re-rendering something if it knows that it hasn't changed, whereas that is impossible for functions like this. But again maybe better to discuss in the CDM

Reply:
Yeah let's maybe add it to the CDM doc then? :) :+1:

Reply:
Added it! To answer your question about using configs, it's just a matter of changing

```
const config = {
  renderSomething: (value: string) => <>{value}</>
}

<>{config.renderSomething('text')}</>
```

To

```
const config = {
  Something: ({ value }: { value: string }) => <>{value}</>
}

<config.Something value="Text" />
```

Reply:
Oh ok! Cool - I didn't know you could do that :) Thanks Luuc!